src/components/NcRichContenteditable/NcRichContenteditable.vue

```diff
@@ -950,9 +950,13 @@ export default {
 		 */
 		showTribute(trigger) {
 			this.focus()
-			const index = this.tribute.collection.findIndex(collection => collection.trigger === trigger)
-			this.tribute.showMenuForCollection(this.$refs.contenteditable, index)
-			this.updateValue(this.$refs.contenteditable.innerHTML)
+			// Focus may be delayed, so we wait for the next frame to show the menu.
+			// Tribute requires caret position (this focus) to set its position
```

Comment:
Maybe something like

```suggestion
			// For an unknown reason in Vue 3 calling `tribute.showMenuForCollection` immediately after `contenteditable.focus()` results in the wrong positioning.
			// Waiting one animation frame for focusing fixes the issue.
```